deep_groove_project/src/first_eda/first_eda.py

```diff
@@ -42,10 +43,10 @@ def show_3d_scatter(df):
     n = df.iloc[:, 1]
     lifetime = df.iloc[:, 2]
 
-    ax.scatter(fr, n, lifetime)
+    ax.scatter(fr, n, np.log(lifetime))
     ax.set_xlabel('Fr')
     ax.set_ylabel('n')
-    ax.set_zlabel('Lifetime')
+    ax.set_zlabel('log(Lifetime)')
```

Comment:
Please use the log with base 10 for all representations of the lifetime.

Reply:
changed the function to log10( )